tests/test_strdist_algorithm.R

```diff
@@ -0,0 +1,99 @@
+library(RPostgreSQL)
+library(assertthat)
+library(magrittr)
+library(readr)
+library(readxl)
+library(dplyr)
+library(stringr)
+
+
+
+# Read test data
+data <- read_csv('data/matching_algorithm_sample.csv') %>% sample_n(1000)
+nctids <- data$'NCT Number'
+# data <- read_xlsx('data/01-foci_weight_assignment.xlsx', sheet = 'test-trials') %>%
+#   select('NCT ID')
+# nctids <- data$'NCT ID'
+
+
+# Check that TRN is well-formed
+assertthat::assert_that(
+  is.character(nctids),
+  all(grepl("^NCT\\d{8}$", nctids))
+)
+
+# Fixed database connection details
+dbname <- "aact"
+host <- "aact-db.ctti-clinicaltrials.org"
+port <- 5432
+
+username <- 'martinholst'
+password <- 'mupmub-tyjcYg-2bykco'
```

Comment:
@Martin-R-H , could you anonymize your credentials, I will then integrate the changes